<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualization</class>
 <widget class="QDialog" name="Visualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>663</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Демонстрация работы</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Задержка отрисовки </string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Запустить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="minimum">
      <number>8</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Ориентированный</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="3">
    <layout class="QFormLayout" name="Draw_Layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </item>
   <item row="2" column="6">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="suffix">
      <string> с</string>
     </property>
     <property name="minimum">
      <double>0.500000000000000</double>
     </property>
     <property name="maximum">
      <double>1.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="baseSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Генерировать вес</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Начальная вершина:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Количество вершин:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string> Таблица результатов поиска</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Открыть теорию</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Пауза</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
